### ISSUE TITLE
[Suggestion] Add an option in search to disable results previewing.

### ISSUE THREAD

**Issue by Haralil** [tryashtar/nbt-studio#38]
Hello.
I recently discovered a problem with my modded world and had to remove a lot of data from region files due to some errors. Each region file in my world has around 6 thousands of bad entries, and opening every NBT tree in NBT Studio takes a really long time. Moreover, when I am deleting the entries, it has to redraw everything again, and I had to waste ~10 hours until I found out that you can just close the trees and do the deletion. Instead of taking 10 minutes for a deletion per file, it now took seconds.
Can you please add a setting so that previewing found entries can be disabled? That would really help when working with large data set with a lot of entries, like world regions.
Thanks.

**Comment by tryashtar** [comment]
I'm really sorry for the bad performance. Can you show me which preview you're referring to?

**Comment by Haralil** [comment]
Yes, sure thing. I am currently editing my world, and when the search finishes, it opens every tree.
![image](https://user-images.githubusercontent.com/16758443/167199495-f157d8b7-bfe3-4fac-b5cf-93b07a7a4bc7.png)
If, instead, after searching, just as shown in the 2nd picture, it would not open every tree with found results, that would improve the speed drastically.
![image](https://user-images.githubusercontent.com/16758443/167199772-e98e3d55-f5df-43ab-9065-39eea2f014f4.png)

**Comment by tryashtar** [comment]
Got it! Thank you

**Comment by tryashtar** [comment]
Am I correct to assume you've managed to build and use your modified copy? :)
Because if so, I'm going to hold off on fixing it myself for a while, until it becomes necessary 😂

**Comment by Haralil** [comment]
Sorry for the late response, yes, I builded it and finished what I was doing. The time improved from ~5 minutes per search to being less than a minute long, approximately.
You can test my fix yourself, if you want, and add it now, but I don't think that's necessary and you'll probably do a better job than me & will probably add an additional setting for this to work better, I just don't feel like dealing with that right now :)

Just wanted to point out a potential fix :)

**Comment by AngelOfCreation** [comment]
Hey can we get this pushed through? It's been 3 years...
